src/Microsoft.AspNet.FileProviders.Common/project.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.0.0-*",
+    "description": "Implementation of ASP.NET 5 file provider abstractions.",
+    "dependencies": {
+        "Microsoft.AspNet.FileProviders.Interfaces": "1.0.0-*"
+    },
```

Comment:
Don't shared source projects need `"shared": "*.cs",` ? Or is this not shared source? The bug didn't say anything about having a package named "Microsoft.AspNet.FileProviders.Common" did it?